spec/lib/pre_assembly/file_manifest_spec.rb

```diff
@@ -90,184 +235,12 @@
   # rubocop:disable Metrics/AbcSize
   def setup_dobj(druid, object, file_manifest)
     allow(bc.batch).to receive(:file_manifest).and_return(file_manifest)
-    PreAssembly::DigitalObject.new(bc.batch, container: object, stager: PreAssembly::CopyStager, dark: false).tap do |dobj|
```

Comment:
so this is a place we no longer can indicate dark?  If we can still indicate it, do we have a test for the behavioral difference?

Reply:
Right, we no longer pass in dark as a parameter to PreAssembly::DigitalObject.  The access rights are set in the StructuralBuilder.  There is no behavioral difference, because these are just the application's models.